avro_to_python/cli.py

```diff
@@ -3,6 +3,7 @@
 """Console script for avro_to_python."""
 import os
 import sys
+sys.path.insert(0, '/Users/ethanschwab/Documents/dev/avro-to-python')
```

Comment:
Assuming this would just be a temp thing (along with the other references to your local env)